FAQ.md

```diff
@@ -162,7 +162,7 @@ way.
 
 ### What is mobile number portability?
 
-The ability to keep your mobile phone number when changing carriers. To see whether a region supports mobile number portability use [isMobileNumberPortableRegion](https://github.com/googlei18n/libphonenumber/blob/master/java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java#L3275).
+The ability to keep your mobile phone number when changing carriers. To see whether a region supports mobile number portability use [isMobileNumberPortableRegion](https://github.com/googlei18n/libphonenumber/blob/master/java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java#L3500).
 
```

Comment:
Hi,
Changing the line number would not work as it could change again in future. So it is better to always use permalink. Could you please copy paste the below link.



```https://github.com/googlei18n/libphonenumber/blob/39b0f4a71a94637407adea4cbda73348b3fb2ce9/java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java#L3500```